third_party/python/pip.bzl

```diff
@@ -62,6 +63,7 @@ def _pip_wheel_impl(rctx):
         "-m",
         "pip",
         "wheel",
+        "--use-pep517",
```

Comment:
I'm not sure why, but this flag ended up being a breaking change, after some timer expired. `pkginfo.get_metadata()` failed to get the metadata from pyfinite 1.9.1's wheel, and that caused the install script to crash. Just a heads up!